spec.html

```diff
@@ -7200,46 +7220,41 @@ <h1>CreateImportBinding ( _N_, _M_, _N2_ )</h1>
       </emu-clause>
     </emu-clause>
 
-    <emu-clause id="sec-lexical-environment-operations">
-      <h1>Lexical Environment Operations</h1>
-      <p>The following abstract operations are used in this specification to operate upon lexical environments:</p>
+    <emu-clause id="sec-environment-record-operations" oldids="sec-lexical-environment-operations">
+      <h1>Environment Record Operations</h1>
+      <p>The following abstract operations are used in this specification to operate upon Environment Records:</p>
 
       <emu-clause id="sec-getidentifierreference" aoid="GetIdentifierReference">
         <h1>GetIdentifierReference ( _lex_, _name_, _strict_ )</h1>
-        <p>The abstract operation GetIdentifierReference takes arguments _lex_ (a Lexical Environment or *null*), _name_ (a String), and _strict_ (a Boolean). It performs the following steps when called:</p>
+        <p>The abstract operation GetIdentifierReference takes arguments _lex_ (an Environment Record or *null*), _name_ (a String), and _strict_ (a Boolean). It performs the following steps when called:</p>
```

Comment:
`lex` is kind of a historical name here now. We use `env` a lot elsewhere; that's probably a better choice now.

Reply:
Yup. Fixing that and also `_thisLex_`.